### PR TITLE
feat(replays): move full replay button

### PR DESCRIPTION
**Pull request by scefali**
Before:
before

After:
after

### CODE REVIEW

File: static/app/components/events/eventReplay/replayPreviewPlayer.tsx [hunk 1]
```diff
@@ -224,4 +221,12 @@ const StyledReplayCell = styled(ReplayCell)`
   padding: 0 0 ${space(1)};
 `;
 
+const HeaderWrapper = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: ${space(1)};
+  margin-bottom: ${space(1)};
```
Comment (ryan953) [inline static/app/components/events/eventReplay/replayPreviewPlayer.tsx:229]: is this margin-bottom needed?

**Comment by michellewzhang** [comment]
should blur the PII in the screenshot!

File: static/app/components/events/eventReplay/replayPreviewPlayer.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/components/events/eventReplay/replayPreviewPlayer.tsx:229]: don't need the `gap` here either

Comment (scefali) [reply static/app/components/events/eventReplay/replayPreviewPlayer.tsx:229]: @ryan953 don't need the gap but the margin bottom makes sure we don't have the button too close to the actual replay player

Comment (ryan953) [reply static/app/components/events/eventReplay/replayPreviewPlayer.tsx:228]: the username is far from the player too now. maybe like `align-items: top` or whatever is what it needs

**Review by michellewzhang — APPROVED** [review]

**Pull request merged**